samples/snippets/requirements.txt

```
@@ -1,6 +1,6 @@
-google-cloud-bigquery-storage==2.26.0
-google-cloud-bigquery==3.25.0
-pandas-gbq==0.23.1
+google-cloud-bigquery-storage==2.27.0
+google-cloud-bigquery==3.27.0
+pandas-gbq==0.24.0
 pandas===2.0.3; python_version == '3.8'
 pandas==2.2.3; python_version >= '3.9'
-pyarrow==17.0.0; python_version >= '3.8'
+pyarrow==18.0.0; python_version >= '3.8'
```

Looks like there is no 18.0.0 release for Python 3.8.

```suggestion
pyarrow==18.0.0; python_version >= '3.9'
```